### PR TITLE
install-and-configure-onlyoffice-docs-for-nextcloud

### PR DESCRIPTION
**Pull request by kseniafedoruk**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Ksenia Fedoruk <kseniya.fedoruk@onlyoffice.com>

### CODE REVIEW

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+The suite can be connected with your `Nextcloud` instance bringing you the ability to edit and co-author office documents right from the `Nextcloud` frontend. 
+
+This tutorial will describe how to install `ONLYOFFICE Document Server` and connect it to your Nextcloud instance via the `integration app`.
+
+**Prerequisites**
+
+*	Ubuntu 20.04 server 
+*	A domain for ONLYOFFICE Document Server (e.g. yourdomain.com)
+*	Your Nextcloud installed on the different machine
```
Comment (mhupfauer) [inline tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:30]: Your Nextcloud installed on **a** different machine

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+
+## Step 1 - Set up Docker repository
+
+The easiest way to start an instance of `ONLYOFFICE Document Server` is to use Docker. Before you install Docker on a new host machine, you need to set up the Docker repository:
+
+```shell=
```
Comment (mhupfauer) [inline tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:37]: $ curl -fsSL https://get.docker.com -o get-docker.sh
$ sh get-docker.sh

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    gnupg \
+    lsb-release
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+echo \
+  "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+```
+
+## Step 2 - Install Docker
```
Comment (mhupfauer) [inline tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:52]: Remove due to above comment, with docker install script and update step numbering

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+sudo apt-get update 
+sudo apt-get install docker-ce docker-ce-cli containerd.io
+```
+
+
+## Step 3 - Install ONLYOFFICE Document Server
+
+Run Docker and execute the following command to install `ONLYOFFICE Document Server` and all the dependencies:
+
+```shell=
```
Comment (mhupfauer) [inline tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:66]: For a persistent installation use a docker-compose.yml file and manage the container with compose.
This enables users to version config files and easily restore them on different machines.

**Review by mhupfauer — CHANGES_REQUESTED** [review]

**Comment by kseniafedoruk** [comment]
Hey! I've made and committed the necessary changes to the article.

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md [hunk 5]
```diff
@@ -27,47 +27,28 @@ This tutorial will describe how to install `ONLYOFFICE Document Server` and conn
...
-
-## Step 3 - Install ONLYOFFICE Document Server
+## Step 2 - Install ONLYOFFICE Document Server
 
 Run Docker and execute the following command to install `ONLYOFFICE Document Server` and all the dependencies:
 
 ```shell=
 sudo docker run -i -t -d -p 80:80 --restart=always onlyoffice/documentserver
 ```
+For a persistent installation use a docker-compose.yml file and manage the container with compose. This enables users to version config files and easily restore them on different machines.
```
Comment (mhupfauer) [inline tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:49]: Add this link: https://github.com/ONLYOFFICE/Docker-DocumentServer/blob/master/docker-compose.yml so people can get easily to the compose file.

**Review by mhupfauer — CHANGES_REQUESTED** [review]
One little change request :)
Otherwise this looks good!

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md (hunk 5 above)
Comment (kseniafedoruk) [reply tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:49]: I've already added (the 2nd commit). And I see the link is available here https://github.com/kseniafedoruk/community-content/blob/master/tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md.

**Comment by kseniafedoruk** [comment]
Thanks! The link is added.

**Review by mhupfauer — APPROVED** [review]

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md (hunk 2 above)
Comment (rook59) [reply tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:37]: Not sure if I like that part.. From https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script

> The convenience script is not recommended for production environments, but can be used as an example to create a provisioning script that is tailored to your needs.

What do you think?

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (rook59) [inline tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:153]: Please sign off here

Comment (kseniafedoruk) [reply tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md]: Done!

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md (hunk 2 above)
Comment (kseniafedoruk) [reply tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:37]: We can get back to installing using the repository. Would it be better?

**Comment by kseniafedoruk** [comment]
Hey! Can we move forward or is there any editing still needed?

File: tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md (hunk 2 above)
Comment (rook59) [reply tutorials/install-and-configure-onlyoffice-docs-for-nextcloud/01.en.md:37]: Both works, I would just prefer the previous one. We can keep it like that.

**Review by rook59 — APPROVED** [review]
Thank you for your submission!

**Pull request merged**